### ISSUE TITLE
ESIMD/api/esimd_pack_unpack_mask.cpp fails on OpenCL backend.

### ISSUE THREAD

**Issue by bader** [intel/llvm-test-suite#901]
GPU driver: 22.09.22577
Full log: https://github.com/intel/llvm/runs/5465039681?check_suite_focus=true
```
Testing N=1, misc...
    ERROR: non-zero lane 1: 0xeb
    ERROR: non-zero lane 4: 0xefe0
    ERROR: non-zero lane 5: 0xeb
    ERROR: non-zero lane 8: 0x30
    ERROR: non-zero lane 12: 0x30
    ERROR: non-zero lane 16: 0xebe0
    ERROR: non-zero lane 17: 0xeb
    ERROR: non-zero lane 20: 0xe630
    ERROR: non-zero lane 21: 0xeb
    ERROR: non-zero lane 24: 0x6163
    ERROR: non-zero lane 25: 0x6c6c
    ERROR: non-zero lane 26: 0x692e
    ERROR: non-zero lane 27: 0x692e
    ERROR: non-zero lane 28: 0x3032
    ERROR: non-zero lane 29: 0x692e
  FAILED
```

**Comment by dongkyunahn-intel** [comment]
ESIMD_EMULATOR backend shows similar failure. Even if `__esimd_pack_mask` and `__esimd_unpack_mask` function implementations return all zero, mismatches from test results do not change.

**Comment by dongkyunahn-intel** [comment]
I included its fix in my PR - https://github.com/intel/llvm-test-suite/pull/913/commits/4ed29c07ec02fea6db3c7b7f46d624afd6a8749c.

**Comment by dongkyunahn-intel** [comment]
esimd_pack_unpack_mask.cpp passes - http://pre-commit.llvm.intel.com:8013/data?file=LLVM-Test-Suite-CI/PR913/beeac65ef2/Linux_LLVM-Test-Suite_SUCCESS.log

**Comment by bader** [comment]
> I included its fix in my PR - [4ed29c0](https://github.com/intel/llvm-test-suite/commit/4ed29c07ec02fea6db3c7b7f46d624afd6a8749c).

Could you provide a link to the PR, please?

**Comment by dongkyunahn-intel** [comment]
> Could you provide a link to the PR, please?

Sure - https://github.com/intel/llvm-test-suite/pull/913. Sorry for confusion.
